ocaml/forkexecd/lib/forkhelpers.ml

```diff
@@ -36,6 +36,10 @@ let test_path =
 
 let runtime_path = Option.value ~default:"/var" test_path
 
+let with_tracing ?traceparent ~name f =
+  let name = "forkhelpers" ^ "." ^ name in
```

Comment:
Avoid using `^` except in the simplest cases; use `Printf` or `String.concat`

Reply:
Done!